src/Integration.Vsix.UnitTests/CFamily/CFamilyIssueToAnalysisIssueConverterTests.cs

```diff
@@ -119,14 +127,136 @@ public void Convert_EndLineIsZero()
 
             // Assert
             issue.StartLine.Should().Be(101);
+            issue.StartLineOffset.Should().Be(0);
 
             issue.EndLine.Should().Be(0);
-            issue.StartLineOffset.Should().Be(0);
             issue.EndLineOffset.Should().Be(0);
+        }
+
+        [TestMethod]
+        public void Convert_LocationEndLineIsNotZero_OffsetsAreSubstructed()
```

Comment:
`Substructed`? Is this a typo of "Subtracted"?
Either way, `OffsetsAreCalculatedCorrectly` might be better.

Reply:
Ah, I knew something was off!